HW3/Task3.py

```diff
@@ -0,0 +1,8 @@
+while True:
+    answer = input('Please enter your answer for this expression - 2+2*2:')
+    if answer == '6':
+        print('You are right')
+        break
+    elif answer != '6':
+        print('You are wrong')
```

Comment:
You can also simplify a bit `elif` statement:
```suggestion
    else:
        print('You are wrong')
```